atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/ThreeNodeCassandraClusterOperations.java

```diff
@@ -66,11 +73,20 @@ private void setReplicationFactorOfSystemAuthenticationKeyspaceToThree()
 
     private boolean systemAuthenticationKeyspaceHasReplicationFactorThree()
             throws IOException, InterruptedException {
-        String output = runCql("SELECT * FROM system.schema_keyspaces;");
-        int replicationFactor = CassandraCliParser.parseSystemAuthReplicationFromCqlsh(output);
+        String getAllKeyspaces = getCqlQueryForCurrentlyRunningCassandra();
+        String output = runCql(getAllKeyspaces);
+        int replicationFactor = cassandraCliParser.parseSystemAuthReplicationFromCqlsh(output);
         return replicationFactor == 3;
     }
 
+    private String getCqlQueryForCurrentlyRunningCassandra() {
+        if (cassandraVersion.equals("3.7")) {
```

Comment:
It's a bit weird that the other one is `startsWith("2.2.")` and this one is `.equals("3.7")`. We should standardise the two. (Or check both values and throw if it isn't one of the two).

Maybe an enum is better where it goes from `2.2.` to `2_2_X` and `3.7` to `3_7`. Then we can have these specific things in there to make it more coherent.